src/renderer/components/message/MessageList.tsx

```diff
@@ -107,6 +107,11 @@ export default function MessageList({
     composerTextarea && composerTextarea.focus()
   }, [chat.id])
 
+  useEffect(() => {
+    const composerTextarea = refComposer.current.childNodes[1]
```

Comment:
I don't quite like the usage of `childNodes[1]`, but well maybe there is no obvoius more elegant solution...